Double Buffering not working

So I have been trying to work on a homebrew game for the nintendo switch, however I came across a problem. Whenever I try to flip or use double buffering on the graphics, it never works. However, when I open **singlebuffering.nro** in the examples the emulator seems to work just fine.

So there are a few ways to fix this:
1. Is double buffering enabled
2. How do i single buffer, since the source code isn't in the examples, its just **singlebuffering.nro**

@endrift Do you mind sending me a copy of the folder you use for yuzu in visual studios? For some reason Cmake is deciding to give me a error:

-------------------------------------------------------------------------------------------

CMake Error at common/CMakeLists.txt:91 (create_target_directory_groups):
  Unknown CMake command "create_target_directory_groups".


CMake Warning (dev) in CMakeLists.txt:
  No cmake_minimum_required command is present.  A line of code such as

    cmake_minimum_required(VERSION 3.11)

  should be added at the top of the file.  The version specified may be lower
  if you wish to support older CMake versions for this project.  For more
  information run "cmake --help-policy CMP0000".
This warning is for project developers.  Use -Wno-dev to suppress it.

--- a/src/core/hle/service/nvflinger/buffer_queue.cpp
+++ b/src/core/hle/service/nvflinger/buffer_queue.cpp
@@ -36,9 +36,7 @@ u32 BufferQueue::DequeueBuffer(u32 pixel_format, u32 width, u32 height) {
             return false;
 
         // Make sure that the parameters match.
-        auto& igbp_buffer = buffer.igbp_buffer;
-        return igbp_buffer.format == pixel_format && igbp_buffer.width == width &&
-               igbp_buffer.height == height;
+        return buffer.igbp_buffer.width == width && buffer.igbp_buffer.height == height;
     });
     if (itr == queue.end()) {
         LOG_CRITICAL(Service_NVDRV, "no free buffers for pixel_format=%d, width=%d, height=%d",

